cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/dataset2/lib/kv/HBaseKVTableTest.java

```diff
@@ -48,10 +50,15 @@
   public static void beforeClass() throws Exception {
     testHBase = new HBaseTestFactory().get();
     testHBase.startHBase();
+    hBaseTableUtil.createNamespaceIfNotExists(testHBase.getHBaseAdmin(), NAMESPACE_ID);
   }
 
   @AfterClass
   public static void afterClass() throws Exception {
+    Pattern cdapTablesPattern = Pattern.compile("cdap_" + NAMESPACE_ID.getId() + ".*");
+    testHBase.getHBaseAdmin().disableTables(cdapTablesPattern);
+    testHBase.getHBaseAdmin().deleteTables(cdapTablesPattern);
+    hBaseTableUtil.deleteNamespaceIfExists(testHBase.getHBaseAdmin(), NAMESPACE_ID);
```

Comment:
Yes, table names use '.'. However, '.' is not a valid character for HBase namespaces. So I had to create the namespace in HBase as ``cdap_[ns-id]``. 

Internally in HBase, a table name is recognized as ``[namespace]:[table-name]``. So the pattern for the string representation is "cdap_[ns-id].*"

In the main code so far, I have removed all occurrences that refer to table names by strings. However, that pattern is still prevalent in tests. I'll create a separate (doesn't have to be 2.8.0) to similarly remove such occurrences and instead use methods from compat modules. We should have methods like ``disableAllTablesInNamespace``, ``deleteAllTablesInNamespace``, etc in the compat modules and do away with this string matching. However, since this pattern right now is used only in tests, I didn't create more APIs in the compat modules yet.